src/server/controllers/sshKeyController.ts

```diff
@@ -0,0 +1,108 @@
+export {};
+
+const exec = require("child_process").execFile;
+
+import { Request, Response, NextFunction } from "express";
+const fetch = require("node-fetch").default;
+const fs = require("fs");
+
+const sshKeyController: any = {};
+
+function execShellCommand(shellCommand: string, args: Array<string>) {
+  return new Promise((resolve, reject) => {
+    exec(
+      shellCommand,
+      args,
+      (error: string, stdout: string, stderr: string) => {
+        if (error) {
+          console.warn(error);
+        }
+        console.warn(stderr);
+        resolve(stdout ? stdout : stderr);
+      }
+    );
+  });
+}
```

Comment:
Duplicated code is also used in gitController.ts. This function will need to be moved to it's own file and imported as a helper function.